src/Symfony/Bridge/Twig/TwigEngine.php

```diff
@@ -75,14 +75,14 @@ public function exists($name)
 
         $loader = $this->environment->getLoader();
 
-        if ($loader instanceof \Twig_ExistsLoaderInterface) {
+        if ($loader instanceof \Twig_ExistsLoaderInterface || method_exists($loader, 'exists')) {
```

Comment:
And this seems to happen. Otherwise, I don't see how #20435 would ever reach the `getSource()` call.